src/password.cpp

```diff
@@ -0,0 +1,45 @@
+#include <fstream>
+#include <iostream>
+#include <stdio.h>
+#include <wordexp.h>
+
+#include "password.h"
+
+const std::string read_password(const char* file) {
+    wordexp_t exp;
+    std::ifstream ifs;
+    std::string pass = "";
+
+    wordexp(file,&exp,0);
+    if (exp.we_wordc > 0) {
+        ifs.open(exp.we_wordv[0]);
+    }
+    wordfree(&exp);
+    if (!ifs.is_open()) {
```

Comment:
Sorry, that code was copy&paste from the original `src/configcontainer.cpp`. Didn't really thought about it, since I didn't want to "touch" to much existing code.